### ISSUE TITLE
[BUG] GPT4All ist broken

### ISSUE THREAD

**Issue by Heezer** [devoxx/DevoxxGenieIDEAPlugin#392]
GPT4All v3.5.2, local API server is enabled, DevoxxGenee 0.4.4 ends up with following exception:
```
java.lang.IllegalArgumentException: No factory for provider: GPT4All
	at com.devoxx.genie.chatmodel.ChatModelProvider.lambda$getFactory$0(ChatModelProvider.java:49)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at com.devoxx.genie.chatmodel.ChatModelProvider.getFactory(ChatModelProvider.java:49)
	at com.devoxx.genie.chatmodel.ChatModelProvider.getStreamingChatLanguageModel(ChatModelProvider.java:40)
	at com.devoxx.genie.util.ChatMessageContextUtil.createContext(ChatMessageContextUtil.java:59)
	at com.devoxx.genie.controller.ActionButtonsPanelController.handlePromptSubmission(ActionButtonsPanelController.java:69)
	at com.devoxx.genie.ui.panel.ActionButtonsPanel.onSubmitPrompt(ActionButtonsPanel.java:221)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2314)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:407)
...
com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:564)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
...
```

**Comment by stephanj** [comment]
Fixed, will be available in upcoming version 0.4.5

Or you can clone the project, build and install this version manually in IDEA.

<img width="693" alt="image" src="https://github.com/user-attachments/assets/dc8209e4-b18c-4558-99ce-d0b31be3655a" />

**Comment by Heezer** [comment]
Thanks a lot for fastest fix I've ever seen 👍
